handlers/bootstrap.go

```diff
@@ -130,6 +130,17 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 
 	volConfig := map[string]struct{}{"/var/run/docker.sock": {}}
 	envVars := []string{"CATTLE_PHYSICAL_HOST_UUID=" + machine.ExternalId}
+	for key, value := range machine.Labels {
+		label := ""
+		switch value.(type) {
```

Comment:
This would be simpler than a switch statement:
```
if val, ok := asdf.(string); ok {
  label = value.(string)
  labelPair := key + "=" + label
  envVars = append(envVars, labelPair)
}
```
unless I'm missing something, which I could be.